go/client/kvstore_api_handler.go

```diff
@@ -86,35 +93,50 @@ func (a *getEntryOptions) Check() error {
 	return nil
 }
 
+func (t *kvStoreAPIHandler) getUsername() (username string, err error) {
+	status, err := t.config.GetCurrentStatus(context.Background(), 0)
+	if err != nil {
+		return username, err
+	}
+	return status.User.Username, nil
+}
+
 func (t *kvStoreAPIHandler) getEntry(ctx context.Context, c Call, w io.Writer) error {
 	var opts getEntryOptions
 	if err := unmarshalOptions(c, &opts); err != nil {
 		return t.encodeErr(c, err, w)
 	}
+	if opts.Team == nil {
+		team, err := t.getUsername()
+		if err != nil {
+			return t.encodeErr(c, err, w)
+		}
+		opts.Team = &team
+	}
```

Comment:
couple of thoughts here. this logic is duplicated a bunch. can we push the resolution of a team from the command into a single place (i.e. here's what the user passed in, give me back a team i can use for these commands) which should be the same for all of these? either on a top-level function or up on the handler? and related, i'd suggest the default team name be `fmt.Sprintf("%s,%s", username, username)`. that's an actual resolvable team as opposed to a special username pretending to be a team.

Reply:
just confirmed that the Handler struct is created every time someone sends a command, so that it is ok to move the "get username" code up into handleV1 :+1:

Reply:
>  i'd suggest the default team name be fmt.Sprintf("%s,%s", username, username). 
sorry that was a mistake!